portfolio/src/main/java/com/google/sps/classes/ChartDataPoint.java

```diff
@@ -0,0 +1,28 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.classes;
+
+/** Represents a chart data point. */
+public class ChartDataPoint {
+  public final int year;
+  public final int budget;
+  public final Integer patents;
+
+  public ChartDataPoint(int year, int budget, Integer patents) {
```

Comment:
is there a reason for using `Integer` for patents instead of `int`?

Reply:
Apparently not. I put it like that because the patents value in the hashmap was an Integer.